src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -13,6 +13,11 @@ namespace Stratis.Bitcoin.Tests.Consensus
 {
     public class ChainedHeaderTreeTest
     {
+        private readonly int PEER_DEFAULT = -1;
```

Comment:
I don't see this guy being used anywhere. If it is supposed to be ID of local node, use the constant that is already defined in CHT itself.

Reply:
Agreed - removed these constants as they should be in CHT & we should have collective agreement on them